Blockzilla/ShortcutView.swift

```diff
@@ -81,14 +81,26 @@ class ShortcutView: UIView {
         }
     }
     
-    @objc private func didLongPress(sender: UILongPressGestureRecognizer) {
-        if sender.state == .began {
-            let feedbackGenerator = UIImpactFeedbackGenerator(style: .medium)
-            feedbackGenerator.prepare()
-            if let shortcut = self.shortcut {
-                CHHapticEngine.capabilitiesForHardware().supportsHaptics ? feedbackGenerator.impactOccurred() : AudioServicesPlaySystemSound(1519)
-                delegate?.shortcutLongPressed(shortcut: shortcut, shortcutView: self)
+}
+
+extension ShortcutView: UIContextMenuInteractionDelegate {
+    
+    func contextMenuInteraction(_ interaction: UIContextMenuInteraction, configurationForMenuAtLocation location: CGPoint) -> UIContextMenuConfiguration? {
+        
+        return UIContextMenuConfiguration(identifier: nil,
+                                          previewProvider: nil,
+                                          actionProvider: { suggestedActions in
+            
+            let removeFromShortcutsAction = UIAction(title: UIConstants.strings.removeFromShortcuts,
+                                                     image: UIImage(named: "icon_shortcuts_remove")) { [weak self] action in
+                let feedbackGenerator = UIImpactFeedbackGenerator(style: .medium)
+                feedbackGenerator.prepare()
+                if let shortcut = self?.shortcut {
+                    CHHapticEngine.capabilitiesForHardware().supportsHaptics ? feedbackGenerator.impactOccurred() : AudioServicesPlaySystemSound(1519)
+                    self?.delegate?.removeFromShortcutsAction(shortcut: shortcut)
+                }
             }
-        }
+            return UIMenu(title: "", children: [removeFromShortcutsAction])
```

Comment:
Minor: I think there's an init without title.

Reply:
Done. Thank you!